web/src/components/molecules/Asset/AssetList/AssetListTable/index.tsx

```diff
@@ -145,6 +146,7 @@ const AssetListTable: React.FC<AssetListTableProps> = ({
         pagination={pagination}
         toolbar={handleToolbarEvents}
         rowSelection={rowSelection}
+        tableStyle={{ overflow: "scroll" }}
```

Comment:
I think @lavalse said in the acceptance test that the table should scroll horizontally if the content gets long, right? So instead of overflow can you use overflowX.

Reply:
Done!